web/src/components/vars.css

```diff
@@ -8,6 +8,7 @@
     --near-black: #4a4a4a;
     --grey: #e7e5e2;
     --darker-grey: #e6e5e3;
+    --disabled-grey: #cacaca;
```

Comment:
This color is the browser's default disabled color grey on buttons, and it looks good to me, where the other already defined greys did not.